docs/workforce-management-framework/upstream-1/master.adoc

```diff
@@ -47,18 +43,23 @@ include::topics/con-datasync-module.adoc[leveloffset=+2]
 
 == Security
 include::topics/con-introducing-securityfeatures.adoc[leveloffset=+2]
+
+== Security {Keycloak}
 include::topics/con-keycloak.adoc[leveloffset=+2]
 include::topics/ref-keycloak.adoc[leveloffset=+3]
 include::topics/con-keycloak-securitystrategy.adoc[leveloffset=+2]
 include::topics/ref-keycloak-securitystrategy.adoc[leveloffset=+3]
 include::topics/pro-keycloak-enablement.adoc[leveloffset=+2]
 include::topics/pro-keycloak-implementation.adoc[leveloffset=+2]
 
+== Security Passport.js
 include::topics/con-passportauth-module.adoc[leveloffset=+2]
 include::topics/con-passportauth-securitystrategy.adoc[leveloffset=+2]
 include::topics/ref-passportauth-securitystrategy.adoc[leveloffset=+3]
 include::topics/pro-passportauth-pointing-to-a-datasource.adoc[leveloffset=+2]
 
+include::topics/con-logging.adoc[leveloffset=+1]
```

Comment:
This is under the security heading.

Reply:
![screen shot 2017-10-11 at 12 35 12 pm](https://user-images.githubusercontent.com/981838/31440851-40e6ec02-ae89-11e7-8901-9f201bccf17f.png)